src/azure-openapi-validator/tests/resources/incomplete-descriptions.json

```diff
@@ -0,0 +1,116 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Microsoft Azure Redis Cache Management API",
+        "description": "Some cool documentation.",
+        "version": "2014-04-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "basePath": "/",
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}": {
+            "delete": {
+                "tags": [
+                    "Redis"
+                ],
+                "operationId": "Redis_Delete",
+                "description": "Deletes a Redis cache.",
+                "x-ms-long-running-operation": true,
+                "parameters": [
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the resource group."
+                    },
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "name.",
+                        "x-ms-client-name": "name"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParamterer"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "description."
+                    },
+                    "204": {
+                        "description": ""
```

Comment:
Hard to enforce since a "perfect" json file is hard to come up with given all our validation rules and we'd have to update every single json in view of any new validation rule we add. Also, something we can address in detail at a later stage.